frontend/app/test/unit/test-utils.ts

```diff
@@ -43,12 +43,14 @@ export function getUrlMethodAndBody(call: [input: string | URL | Request, init?:
   return { url, method, body };
 }
 
-export async function userTypeInputs(user: UserEvent, inputs: { [key: string]: string | number }) {
+export async function userTypeInputs(user: UserEvent, inputs: { [key: string]: string | number }, skipCheck?: boolean) {
   for (const [key, value] of Object.entries(inputs)) {
     const input = await screen.findByTestId(key);
     await user.clear(input);
     await user.type(input, value.toString());
-    expect(input).toHaveValue(value.toString());
+    if (!skipCheck) {
+      expect(input).toHaveValue(value.toString());
+    }
```

Comment:
The `skipCheck` is not a good solution and not what was suggsted in Mattermost.

More importantly, we should use locators based on roles, not on ids when possible. That wasn't possible for data entry (it is now I just learned), hence the usage of ids there.

So for this PR the locators for polling stations need to be changed. I can pick up the data entry changes in a new PR.

Reply:
yes, i made this change before your suggestion, I will update the PR with a the suggested workaround.

Reply:
stopped using the util and used roleBased selectors